spec/features/locales_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe 'locale' do
+  
+  before {
+    visit root_path(:en)
+  }
+
+  context 'when arabic is selected' do 
+    
+    it 'loads page' do
```

Comment:
`'when arabic is selected it loads page'` does not read like a well-constructed, informative piece of English that reflects a feature of the app.